odbc/src/api/functions.rs

```diff
@@ -1,8 +1,9 @@
 use crate::{
     api::{
         data::{
-            get_diag_rec, input_wtext_to_string, set_output_wstring, set_str_length,
-            unsupported_function,
+            get_diag_rec,
+            i16_len::{set_output_string, set_output_wstring},
```

Comment:
```suggestion
            i16_len,
```
Since we will eventually need the `i32_len` variants of these functions in this file, should we import this as simply `i16_len` and write `i16_len::set_output_[w]string` wherever we use these functions?